.github/workflows/nextpnr.yml

```diff
@@ -46,12 +46,14 @@ jobs:
     - run: dockerBuild nextpnr:ice40      nextpnr ice40
     - run: dockerBuild nextpnr:icestorm   nextpnr icestorm
     - run: dockerBuild nextpnr:ecp5       nextpnr ecp5
+    - run: dockerBuild nextpnr:gowin      nextpnr gowin
```

Comment:
Please, place gowin below prjtrellis. Those are couples:

- ice40 and icestorm
- ecp5 and prjtrellis
- gowin and apicula